tools/Makefile

```diff
@@ -128,6 +129,7 @@ $(curdir)/quilt/compile := $(curdir)/autoconf/compile $(curdir)/findutils/compil
 $(curdir)/sdcc/compile := $(curdir)/bison/compile
 $(curdir)/squashfs3-lzma/compile := $(curdir)/lzma-old/compile
 $(curdir)/squashfs4/compile := $(curdir)/xz/compile $(curdir)/zlib/compile
+$(curdir)/util-linux/compile := $(curdir)/bison/compile
```

Comment:
curious if it makes more sense to build with meson.

Reply:
> curious if it makes more sense to build with meson.

I used meson at first, but it turns out that some components can't be disabled by meson and it finally lead to a build failure in buildbot container.